examples/examples/futures-spawn.rs

```diff
@@ -32,10 +32,11 @@ fn subtask(number: usize) -> impl Future<Item = usize, Error = ()> {
 }
 
 fn main() {
-    use tracing_subscriber::fmt;
-    let subscriber = fmt::Subscriber::builder()
-        .with_max_level(Level::TRACE)
-        .finish();
-    let _ = tracing::subscriber::set_global_default(subscriber);
+    use tracing_subscriber::{fmt, EnvFilter};
+
+    fmt::Subscriber::builder()
+        .with_env_filter(EnvFilter::try_new("futures-spawn=trace").unwrap())
+        .init();
```

Comment:
again, the new example code seems much nicer but I'd prefer it to be in a separate diff